README.dotnet.md

```diff
@@ -0,0 +1,133 @@
+# SpacetimeDB SDK for C#
```

Comment:
it looks like this overlaps a lot with the other `README.md`? Is that on purpose?

Reply:
Yeah, they always were very similar minus a few API changes. We could probably refactor it, but I'm not sure if it's going to make it more confusing to the users to jump between 2 separate docs, so I just kept READMEs from the 2 repos mostly intact.